### PR TITLE
fix NPE on publish test button

### PR DESCRIPTION
**Pull request by zarend**

### CODE REVIEW

File: implementation/src/main/java/com/teampc/controller/test/TestRowController.java [hunk 1]
```diff
@@ -84,7 +84,8 @@ void onEditActionHandler(ActionEvent event) {
    @FXML
    /** Button click handler **/
    void onPublishActionHandler(ActionEvent event) {
-      TestDAO.getInstance().updateTest(new Test());
+      this.test.publish();
+      TestDAO.getInstance().insert(this.test);
```
Comment (adufrene) [inline implementation/src/main/java/com/teampc/controller/test/TestRowController.java]: This will create a new test instead of updating the old one

Comment (adufrene) [inline implementation/src/main/java/com/teampc/controller/test/TestRowController.java]: we'll have two tests after

Comment (Epiphane) [inline implementation/src/main/java/com/teampc/controller/test/TestRowController.java]: @zarend wanna fix this?

Comment (zarend) [inline implementation/src/main/java/com/teampc/controller/test/TestRowController.java]: @thomassteinke, yay, I'll take that

Comment (zarend) [inline implementation/src/main/java/com/teampc/controller/test/TestRowController.java]: @Epiphane ^

Comment (Epiphane) [inline implementation/src/main/java/com/teampc/controller/test/TestRowController.java]: lolol whats up @thomassteinke, welcome to the team

Comment (Epiphane) [inline implementation/src/main/java/com/teampc/controller/test/TestRowController.java]: Crap, that's my account. Oops

Comment (adufrene) [inline implementation/src/main/java/com/teampc/controller/test/TestRowController.java]: we now have over twice the number of comments than lines changed in this pr

**Comment by jamesonrli** [comment]
closing... fixed in #46

**Pull request closed without merging**